src/components/Combobox/Combobox.jsx

```diff
@@ -128,7 +128,8 @@ const Combobox = forwardRef(
       focusedElementRole: useActiveDescendantListFocus.roles.COMBOBOX,
       itemsIds: filteredOptionsIds,
       onItemClick: overrideOnClick,
-      isItemSelectable: isChildSelectable
+      isItemSelectable: isChildSelectable,
+      isIgnoreSpaceAsItemSelection: true
```

Comment:
in combobox we do not want to listen to space event because it can be part of typing the name of the option we searching and writing in the search component (of the combobox)